src/components/sequencing/SequenceEditor.svelte

```diff
@@ -46,6 +52,27 @@
     typescriptDefaults.setExtraLibs(commandDictionaryTsFiles);
   }
 
+  // Update the custom worker to use the select Command Dictionary.
+  $: if (monaco !== undefined && commandDictionaryJson !== undefined) {
+    monaco.languages.typescript
```

Comment:
@Cobular or @goetzrrGit What is the reason we need this entire reactive statement? Does the `fullyLoaded` function not accomplish the same thing?